dvc/output.py

```diff
@@ -583,6 +583,7 @@ def _checkout(self, *args, **kwargs):
         from dvc_data.checkout import CheckoutError as _CheckoutError
         from dvc_data.checkout import LinkError, PromptError
 
+        kwargs.setdefault("ignore", self.dvcignore)
```

Comment:
Only two places are calling this `_checkout` method. One in line `583` had already had the `ignore` argument, and another one is in line `785` in this file. I think we should either remove both of them and add a default value to `ignore` or remove the default one but pass them explicitly in the calls.